Sources/RInAppMessaging/Models/Responses/CampaignData.swift

```diff
@@ -9,7 +9,7 @@ internal struct CampaignData: Codable, Hashable {
     let isCampaignDismissable: Bool
     let messagePayload: MessagePayload
 
-    var intervalBetweenDisplaysInMS: Int? {
```

Comment:
Optional removed because the `messagePayload.messageSettings.displaySettings.delay` will always have value and never be nil.